### PR TITLE
fixes popcorn hair not working for female diona 

### PR DESCRIPTION
**Pull request by salamandooo**
I accidentally fucked something up so now I'm fixing it

### CODE REVIEW

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -858,6 +858,8 @@
 	icon_state = "hair_popcorn"
 	species_allowed = list("Diona")
 	do_colouration = 0
+	gender = FEMALE
+	gender = MALE
```
Comment (boy2mantwicethefam) [inline code/modules/mob/new_player/sprite_accessories.dm:862]: what

Comment (salamandooo) [reply code/modules/mob/new_player/sprite_accessories.dm:862]: it looks like it shouldn't work but it does

Comment (boy2mantwicethefam) [reply code/modules/mob/new_player/sprite_accessories.dm:862]: female diona cant use it

Comment (salamandooo) [reply code/modules/mob/new_player/sprite_accessories.dm:862]: yes they can

Comment (Kurfursten) [reply code/modules/mob/new_player/sprite_accessories.dm:862]: this definitely only sets `gender = MALE`, it's the last one that matters

**Review by ShiftyRail — CHANGES_REQUESTED** [review]
Only the last `gender` thing is taken into account. Please fix that.

**Pull request closed without merging**